virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -98,12 +98,9 @@ See xref:../../architecture/architecture-rhcos.adoc#rhcos-about_architecture-rhc
 
 * {rh-storage-first}: You must create a dedicated storage class for Windows virtual machine disks. See link:https://access.redhat.com/articles/6978371[Optimizing ODF PersistentVolumes for Windows VMs] for details.
 
-[NOTE]
-====
-You must specify a default storage class for the cluster. See xref:../../storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc#persistent-storage-csi-sc-manage[Managing the default storage class]. If the default storage class provisioner supports the `ReadWriteMany` (RWX) xref:../../storage/understanding-persistent-storage.adoc#pv-access-modes_understanding-persistent-storage[access mode], use the RWX mode for the associated persistent volumes for optimal performance.
+* You must create a default {VirtProductName} or {product-title} storage class. If both {VirtProductName} and {product-title} default storage classes exist, the {VirtProductName} class takes precedence when creating VM disks.
```

Comment:
Should we mention how the annotation for the default virt storage class looks like?
There's a Design doc: https://github.com/kubevirt/community/blob/main/design-proposals/default-virt-storage-class.md#design

Reply:
good idea. added that